guides/doc-Planning_Guide/topics/Deployment_Considerations.adoc

```diff
@@ -35,7 +35,7 @@ If you plan to have more than one Red{nbsp}Hat{nbsp}Network account, or if you w
 A customer that does not have a {Project} subscription can create a Subscription Asset Manager manifest, which can be used with {Project}, if they have other valid subscriptions.
 You can then use the multiple manifests in one {ProjectServer} to manage multiple organizations.
 
-If you must manage systems but do not have access to the subscriptions for the RPMs, you must use Red Hat Enterprise Linux Smart Management Add-On.
+If you must manage systems but do not have access to the subscriptions for the RPMs, you must use {RHEL} Smart Management Add-On.
```

Comment:
Should this be "Satellite only"? Or we keep "Red Hat Enterprise Linux" because there is no "Rocky Linux Smart Management Add-On"?

I prefer `ifdef::satellite[]`.

Reply:
Probably, but `{RHEL}` is just a short hand for the written out version. So IMHO it should be a follow up patch.